src/gui/spaces/spacesmodel.cpp

```diff
@@ -129,13 +129,16 @@ QVariant SpacesModel::data(const QModelIndex &index, int role) const
             });
             job->start();
             return _images[item.getId()];
-        } break;
+        }
+        default:
+            return {};
         }
     case Qt::SizeHintRole: {
         switch (column) {
-        case Columns::Image: {
+        case Columns::Image:
             return ImageSizeC;
-        }
+        default:
+            return {};
```

Comment:
Since all branches have been handled explicitly, would it make sense to replace the final `return {}` with `Q_UNREACHABLE()`?

Reply:
It's the default for the `switch` over the column, not the roles.

Reply:
Nevermind, I misinterpreted the code.